code/controllers/subsystems/DPC.dm

```diff
@@ -0,0 +1,55 @@
+/*
+This is pretty much just an optimization for wait=0 timers. They're relatively common, but generally don't actually need the more
+ complex features of SStimer. SSdpc can handle these timers instead (and it's a lot simpler than SStimer is), but it can't handle
+ complex timers with flags. This doesn't need to be explicitly used, eligible timers are automatically converted.
+*/
+
+SUBSYSTEM_DEF(dpc)
+	name = "Delayed Procedure Call"
+	wait = 1
+	runlevels = RUNLEVELS_ALL
+	priority = SS_PRIORITY_DPC
+	flags = SS_TICKER | SS_NO_INIT
+
+	var/list/queued_calls = list()
+	var/avg = 0
+	var/list/unique_queued_calls = list()
+	var/unique_avg = 0
+
+/datum/controller/subsystem/dpc/stat_entry()
+	return ..() + " Q: [queued_calls.len], AQ: ~[round(avg)], UQ: [unique_queued_calls.len], AQ: ~[round(unique_avg)]"
```

Comment:
Nit: might be better to differentiate the two AQ values here, like one of
`N: 0 (Avg: 0) U: 0 (Avg: 0)`
`Q: 0, AQ: 0, UQ: 0, UAQ: 0`

Reply:
I swear I put UAQ?? Huh